docs/src/index.md

```diff
@@ -1,4 +1,31 @@
-# How to guide
+# Thermodynamics
+
+## Principal design
+
+This package uses an abstraction that leverages the idea of a thermodynamic state:
+
+ - given two (or more) independent intrinsic thermodynamic properties, we can establish a thermodynamic state and
+ - given a thermodynamic state, we can compute any thermodynamic property
+
+ 
+## Simple example
```

Comment:
Should we delete some things from Usage section now? Seems repeated?

Reply:
Thanks for pointing that out. I'm going to leave that section but rename it, I think it's helpful to see it's use in a sort of dycore-context.